### ISSUE TITLE
Selfoss stops working after PHP upgrade in Ubuntu 16.04

### ISSUE THREAD

**Issue by hsanson** [SSilence/selfoss#789]
I upgraded my server (apt-get upgrade) that resulted in PHP being upgraded from 7.0.4-7ubuntu2.1 to 7.0.8-0ubuntu0.16.04.1. 

After PHP was upgraded selfoss (2.14) stopped working. I tried upgrading to 2.15 but the error persisted. 

The only way to get selfoss working again was to downgrade PHP to the previous version.

The logs show the following error:

```
[error] 8113#8113: *77 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot use $this as parameter in /home/ryujin/selfoss-2.15/libs/f3/base.php on line 1185" while reading response header from upstream
```

This upgrade also resulted in the rss feed update returning error 500 (even after reverting) but that is another issue #788 .

**Comment by hsanson** [comment]
For the anyone with the same issue this is how I downgraded PHP to make selfoss work again:

```
sudo apt-get install php7.0-fpm=7.0.4-7ubuntu2.1 php7.0-common=7.0.4-7ubuntu2.1 php7.0-cli=7.0.4-7ubuntu2.1 php7.0-json=7.0.4-7ubuntu2.1 php7.0-opcache=7.0.4-7ubuntu2.1 php7.0-readline=7.0.4-7ubuntu2.1 php7.0-sqlite3=7.0.4-7ubuntu2.1 php7.0-mbstring=7.0.4-7ubuntu2.1 php7.0-gd=7.0.4-7ubuntu2.1 php7.0-curl=7.0.4-7ubuntu2.1
```

Note that this does not fix the rss update issue (#788).

**Comment by hsanson** [comment]
Just found this is the same issue as #767 and that there is a fix in PR #768. Any timeline for when this is going to be merged to master?
